src-tauri/src/util.rs

```diff
@@ -64,6 +64,27 @@ pub async fn get_server_player_count() -> Result<(i32, usize), String> {
     Ok((total_player_count, server_count))
 }
 
+#[tauri::command]
+pub async fn kill_northstar() -> Result<(), String> {
+    if !check_northstar_running() {
+        return Err("Northstar is not running".to_string());
+    }
+
+    let s = sysinfo::System::new_all();
+
+    for process in s.processes_by_exact_name("Titanfall2.exe") {
+        log::info!("Killing Process {}", process.pid());
+        process.kill();
+    }
+
+    for process in s.processes_by_exact_name("NorthstarLauncher.exe") {
```

Comment:
Kinda related to #284: can we make sure this will not kill server instead of game client?

Reply:
But how would we do that?
NorthstarLauncher only uses the flag to determine if it should load stubs or not and the wsock32_proxy can be ran via Titanfall2.exe

I guess we could get the whole command line and check if it contains `-dedicated`?
If we do that we should put that into a helper function that takes `sysinfo::Process` to re-use for checking if its running and killing.

Since FlightCore does not differentiate between server and client as is right now, I think its fine to keep this and change it up in a follow-up PR

---

What also just came into my head, we would also need to change any launch method to use `-multiple`, since by default Source grabs a named handle to prevent multiple instances from running.

Reply:
Yeah, I'd skip that tbh. This is on DevView anyway and we primarily target installs with only client running.